src/views/Home/components/TotalValueLockedCard.tsx

```diff
@@ -13,26 +13,24 @@ const StyledTotalValueLockedCard = styled(Card)`
 const TotalValueLockedCard = () => {
   const TranslateString = useI18n()
   const data = useGetStats()
-  const tvl = data ? data.total_value_locked_all.toLocaleString('en-US', { maximumFractionDigits: 2 }) : null
+  const tvl = data ? data.total_value_locked_all.toLocaleString('en-US', { maximumFractionDigits: 0 }) : null
 
   return (
     <StyledTotalValueLockedCard>
       <CardBody>
-        <div>
-          <Heading size="lg" mb="24px">
-            {TranslateString(999, 'Total Value Locked (TVL)')}
-          </Heading>
-          {data ? (
-            <>
-              <Heading size="xl">{`$${tvl}`}</Heading>
-              <Text color="textSubtle">{TranslateString(999, 'Across all LPs and Syrup Pools')}</Text>
-            </>
-          ) : (
-            <>
-              <Skeleton height={66} />
-            </>
-          )}
-        </div>
+        <Heading size="lg" mb="24px">
+          {TranslateString(999, 'Total Value Locked (TVL)')}
+        </Heading>
```

Comment:
This needs to wrap. On 320px it is overflowing.
![image](https://user-images.githubusercontent.com/72719739/105126486-2c965f00-5b22-11eb-8748-36d804cb7d88.png)

Reply:
![image](https://user-images.githubusercontent.com/72658581/105126681-a2022f80-5b22-11eb-8c4d-604da88fa80c.png)
all good for me